### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_Connect to a MySQL Server_I want to connect to a MySQL Server by providing server and authentication details so that I can manage it_VoiceOver: Voiceover is not announcing the "Connection Details" section name when voiceover focus is on the first field of section.

### ISSUE THREAD

**Issue by Raghu-addepalli** [microsoft/azuredatastudio#22680]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#Screenreader; #VoiceOver; #Benchmark; #WCAG1.3.1; #Info and Relationships; #Benchmark; #FTP; #MAC; #AILimited; #Element:Dropdown; #A11yWCAG2.1;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Environment and OS details:
Application Name: Azure Data Studio
Application Version: 14.2.0
Operating system: macOS Ventura (Version 13.3)
Screen Reader: Voiceover.

### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till new connection and press enter
4. Enter a valid server name username and password and press connect

### Repro Steps:

1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Connection' icon.
4. Select server type as MySQL.
5. Turn on voiceover.
6. Tab to "Connection type" dropdown present in the "Connection Details" section.
7. Now check that Voice over is announcing the section name or not.

### Actual Result:
Voiceover is not announcing the "Connection Details" section name when voiceover focus is on the first field of section.

### Expected Result:
Voiceover is not announcing the "Connection Details" section name when voiceover focus is on the first field of section.

### User Impact:
Screen Reader users will not be able to understand in which section the focus is there if the section name is not announced.

### Attachment:

https://user-images.githubusercontent.com/91586743/231141140-f7329945-6739-4821-82e3-f7bcb82734f0.mp4

**Comment by Raghu-addepalli** [comment]
Primary bug - https://github.com/microsoft/azuredatastudio-mysql/issues/197

**Comment by alanrenmsft** [comment]
@Raghu-addepalli  Please use the latest insider build to verify the fix.

**Comment by alanrenmsft** [comment]
@Raghu-addepalli looks like you are using the stable build, you should use the latest insiders build as I mentioned in my comments. https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch

**Comment by Raghu-addepalli** [comment]
verified the issue in below version, issue is Fixed Hence closing the Bug and added attachment.

Version details:
Version: 1.45.0-insider (Universal)
Commit: f7bc38d552e2cfbd7e8cadcf6cb40ed5f7cf9066
Date: 2023-05-09T05:57:43.872Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin x64 22.4.0

[22680_Fixed.zip](https://github.com/microsoft/azuredatastudio/files/11438731/22680_Fixed.zip)
